recipes/ringdecomposerlib/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "RingDecomposerLib" %}
+{% set version = "1.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/rareylab/RingDecomposerLib/archive/refs/tags/v{{ version }}_rdkit.tar.gz
+  sha256: 944b5816712a48bbf88aa25d4300ce11871ddf6e971218eac08f90ed2192f715
+  patches:
+    - language_c.patch
+
+build:
+  number: 0
+
+  run_exports:
+    - {{ pin_subpackage(name|lower, max_pin='x') }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - cmake
+    - ninja
+
+test:
+  commands:
+    - test -f $PREFIX/lib/lib{{ name }}${SHLIB_EXT}  # [unix]
+    - if not exist %LIBRARY_PREFIX%\\bin\\{{ name }}.dll exit 1  # [win]
+    - if not exist %LIBRARY_PREFIX%\\lib\\{{ name }}.lib exit 1  # [win]
```

```suggestion
    - test -f $PREFIX/lib/lib{{ name }}${SHLIB_EXT}  # [unix]
    - ! test -f $PREFIX/lib/lib{{ name }}Static.a  # [unix]
    - if not exist %LIBRARY_PREFIX%\\bin\\{{ name }}.dll exit 1  # [win]
    - if not exist %LIBRARY_PREFIX%\\lib\\{{ name }}.lib exit 1  # [win]
    - if exist %LIBRARY_PREFIX%\\lib\\{{ name }}Static.lib exit 1  # [win]
```
Your outputs contain static libraries; these are not allowed. Please test that the static libs are excluded from the outputs. You can either add a call to `rm` as the last line in the build scripts or patch the CMake files/options so these are not generated. Hopefully, there is already a CMake option to skip building the static libs in the first place.

https://github.com/conda-forge/cfep/blob/main/cfep-18.md